kwok/commands.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+kwokctl create cluster --name=kwok
+kwokctl get clusters
+kubectl cluster-info --context kwok-kwok
+oc get namespace
```

Comment:
Why do we need to do this as part of the image build?

Reply:
Im just showcasing here the oc cmd works. that's also part of AC.

Reply:
yeah, but a showcase should not become a permanent part of the image...

Reply:
removed